src/pylorax/api/v0.py

```diff
@@ -998,6 +999,28 @@ def take_limits(iterable, offset, limit):
     """
     return iterable[offset:][:limit]
 
+# A decorator for checking the parameters provided to the API route implementing
```

Comment:
This should go into a library module so that the future v1.py can use it as well.